### PR TITLE
feat(grunt-bower-release): support variety of options for tagging versions

### PR DESCRIPTION
**Pull request by mikeborozdin**
This is a README.MD featuring the new configuration options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,8 +79,13 @@ It's super simple stuff, and I hope some people might find it useful!
 3. `stageDir` -- A staging directory where the repository is built and tagged.
 4. `main` -- Enables the grunt task to override `bower.json`'s `main` parameter. This will be ignored if it is not a string or an array.
 5. `dependencies` -- Enables the grunt task to add dependencies to a build.
-6. `extendDependencies` -- If true, the depencies from the source repository will be extended with the grunt options dependencies.
+6. `extendDependencies` -- If true, the dependencies from the source repository will be extended with the grunt options dependencies.
```
Comment (caitp) [inline README.md:82]: oops /).(\

File: README.md [hunk 2]
```diff
@@ -79,8 +79,13 @@ It's super simple stuff, and I hope some people might find it useful!
...
 7. `branchName` -- (Optional) Specify the branch used for the endpoint.
+8. ```overwriteTag``` -- (Optional) If true, a Git tag will be overwritten. The plug-in creates a Git tag in order to specify a package version for ```Bower```.
```
Comment (caitp) [inline README.md:84]: I thought this was called `overwriteVersion` when I merged this, but I guess I was wrong. It's fine though, lgtm

**Comment by mikeborozdin** [comment]
@caitp,

I wonder if we are going to publish a new version to the central NPM registry. At the moment, it still has an old one - https://www.npmjs.org/package/grunt-bower-release

**Comment by caitp** [comment]
Yes --- but not right away. In the mean time, you can get the latest version by running `npm install https://github.com/caitp/grunt-bower-release/tarball/master`

**Comment by mikeborozdin** [comment]
Cool. At the moment being we have to keep latest version in our private NPM registry, but ideally we would love to switch to the one that is in the central NPM registry.

**Comment by mikeborozdin** [comment]
@caitp,

Do you think it will make into the central NPM registry anytime soon?

**Pull request closed without merging**